dark/aa.py

```diff
@@ -409,6 +411,257 @@
 }
 
 
+"""
+Clusters based on raw amino acid property values. See
+https://notebooks.antigenic-cartography.org/barbara/pages/features/
+aa-properties.html and https://notebooks.antigenic-cartography.org/barbara/
+pages/features/new-tps.html
+"""
+
+PROPERTY_CLUSTERS = {
+    'A': {
+        'aliphaticity': 1,
+        'aromaticity': 1,
+        'composition': 1,
+        'hydrogenation': 1,
+        'hydropathy': 3,
+        'hydroxyethilation': 2,
+        'iep': 2,
+        'polar_req': 2,
```

Comment:
It's the abbreviation for 'polar requirement'. I can change it to `polar_requirement`.